src/components/Donation/components/CardForm.js

```diff
@@ -35,15 +51,22 @@ class CardForm extends PureComponent {
 
   render() {
     const { amount } = this.props;
-    const { isFormValid } = this.state;
+    const { isFormValid, isCaptchaReady } = this.state;
     return (
       <Form className='donation-form' onSubmit={this.submit}>
         <StripCardForm getValidationState={this.getValidationState} />
+        <Reaptcha
+          onLoad={this.onLoad}
+          onVerify={this.onVerify}
+          ref={e => (this.captcha = e)}
```

Comment:
[`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) is the preferred way to use `refs` in react now.

The `reapture` lib exports a class component, so the `createRef` method is ok.